src/GridFieldOrderableRows.php

```diff
@@ -359,7 +360,15 @@ public function getManipulatedData(GridField $grid, SS_List $list)
                 // Fix bug in 3.1.3+ where ArrayList doesn't account for quotes
                 $sortterm .= $this->getSortTable($list).'.'.$this->getSortField();
             } else {
+                // If not an ArrayList it's definitely a DataList right?
+                // Or do we need another conditional here?
                 $sortterm .= '"'.$this->getSortTable($list).'"."'.$this->getSortField().'"';
+                $classname = $list->dataClass(); // DataLists have a dataClass
+                if (Config::inst()->get($list->dataClass(), 'default_sort')) {
+                    // Append the default sort to the end of the sort string
+                    // This may result in redundancy... but it seems to work
+                    $sortterm .= ', ' . Config::inst()->get($classname, 'default_sort');
```

Comment:
`$sorttterm` can be an empty string here, so we can't start with the comma here, will need further logic.

Reply:
Adjusted to only add a comma  when the string is not empty.